lib/get-dialect-ids.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from "vitest";
+import { getDialectIds, loadDialect } from "./keywords.js";
+import "../draft-2020-12";
+import "../draft-2019-09";
+
+
+test("should return only imported schema in the array array if no custom dialects are loaded", () => {
+  const dialectIds = getDialectIds();
+  expect(dialectIds).toEqual(["https://json-schema.org/draft/2020-12/schema"]);
+});
+
+test("should return an array of dialect identifiers", () => {
+  //Load some dialects before each test
+  loadDialect("http://example.com/dialect1", {
+    "https://json-schema.org/draft/2020-12/vocab/core": true,
+    "https://json-schema.org/draft/2020-12/vocab/applicator": true
+  });
+  loadDialect("http://example.com/dialect2", {
+    "https://json-schema.org/draft/2020-12/vocab/core": true,
+    "https://json-schema.org/draft/2020-12/vocab/applicator": true
+  });
+  const dialectIds = getDialectIds();
+  expect(dialectIds).toEqual([
+    "https://json-schema.org/draft/2020-12/schema",
+    "http://example.com/dialect1",
+    "http://example.com/dialect2"
+  ]);
+});
```

Comment:
Wrap all these tests in a `describe`. It makes the test output nicer to consume. It should be something that indicates that these tests are related to dialects, vocabularies, and keywords. Keep it as simple as possible.